mineral-ui/mineral-ui
# Update all eslint and prettier dependencies

### Description

* Update all eslint and prettier dependencies
* Reformat code following prettier update

### Motivation and context

We were falling behind on these dependencies.  Note that I intentionally did not enable the trailing comma option.  I tried it out and I still hate it.

closes #250 
closes #126 
closes #116 
closes #108 
closes #104
closes #98

### How to test

There should be no eslint errors.  Code formatting should be slightly prettier.  There should be no functional differences.  FYI, I had to update atom and restart to get rid of some eslint errors.

### Types of changes

- Other (provide details below)

Updated dependencies and slightly different code formatting.

### Checklist


### How does this PR make you feel?

![fancy](https://media.giphy.com/media/HPpatGk9a7bDq/giphy.gif)
